cxxheaderparser/parser.py

```diff
@@ -1655,6 +1664,7 @@ def _parse_function(
         destructor: bool,
         is_friend: bool,
         is_typedef: bool,
+        msvc_convention: typing.Optional[LexToken],
```

Comment:
At some point it'll probably make more sense to have a 'parse context' to pass along instead of all of these arguments...